OpenAPI/kiota/using.md

```diff
@@ -9,6 +9,18 @@ ms.date: 03/10/2023
 
 # Using the Kiota tool
 
+Kiota now supports the generation of AI plugins, including Type-B Plugin manifests, tailored for Microsoft Copilots. These AI plugins can be used to ground Large Language Models (LLMs) by integrating them with relevant, use-case-specific data, thereby enhancing the user experience.
+
+This article provides instructions on how to use Kiota in Visual Studio Code (VS Code) to generate AI plugins, create API manifests, and integrate them with Microsoft Copilot workflows.
```

Comment:
This documentation page was originally dedicated to the CLI options. It'll need to be revamped once we remove the feature flag for the CLI experience.

I suggest we move the documentation for the VS Code extension to a new dedicated page to avoid mixing things, plus this page is already quite long.

Reply:
I appreciate the input. I'll create a new topic.